src/index.js

```diff
@@ -1,92 +1,45 @@
 /* eslint-disable no-labels */
 
-const FS = require('fs');
-const Path = require('path');
-const EventEmitter = require('events');
-
-const { PNG } = require('pngjs');
-const JPEG = require('jpeg-js');
-const BMP = require('bmp-js');
-const GIF = require('omggif');
-const UTIF = require('utif');
-const MIME = require('mime');
-const tinyColor = require('tinycolor2');
-const BigNumber = require('bignumber.js');
-const bMFont = require('load-bmfont');
-const MkDirP = require('mkdirp');
-const rawBody = require('raw-body');
-const fileType = require('file-type');
-const pixelMatch = require('pixelmatch');
-const EXIFParser = require('exif-parser');
-// polyfill Promise for Node < 0.12
-const Promise = global.Promise || require('es6-promise').Promise;
-
-const ImagePHash = require('./phash.js');
-const request = require('./src/request');
-const Resize = require('./resize.js');
-const Resize2 = require('./resize2.js');
+import FS from 'fs';
+import Path from 'path';
+import EventEmitter from 'events';
+
+import { PNG } from 'pngjs';
+import JPEG from 'jpeg-js';
+import BMP from 'bmp-js';
+import GIF from 'omggif';
+import UTIF from 'utif';
+import MIME from 'mime';
+import tinyColor from 'tinycolor2';
+import BigNumber from 'bignumber.js';
+import bMFont from 'load-bmfont';
+import MkDirP from 'mkdirp';
+import rawBody from 'raw-body';
+import fileType from 'file-type';
+import pixelMatch from 'pixelmatch';
+import EXIFParser from 'exif-parser';
+import sourceMaps from 'source-map-support';
```

Comment:
This should not be in the src file, since this means that we force users to have this dependency for sourcemaps, but for the browser usecase, it is already handled by the browser.

Reply:
This npm module works with browserify. The readme alludes to it.